yarn.lock

```diff
@@ -4,45 +4,50 @@
 
 "@algolia/autocomplete-core@1.2.2":
   version "1.2.2"
-  resolved "https://registry.yarnpkg.com/@algolia/autocomplete-core/-/autocomplete-core-1.2.2.tgz#c121e70c78fd0175c989a219918124ad7758e48b"
+  resolved "https://registry.npmjs.org/@algolia/autocomplete-core/-/autocomplete-core-1.2.2.tgz"
```

Comment:
It looks like you've mistakenly run `npm install` instead of `yarn`.
Please:
- remove `package-lock.json` file
- restore `yarn.lock` file
- use only `yarn` package manager for this project (it's in the 
https://github.com/reactjs/uk.react.dev/blob/main/README.md)

Reply:
Done